src/modules/dataTypes.js

```diff
@@ -1,15 +1,23 @@
 import i18n from '../locales/index.js'
 
-export const ALL_ID = 'ALL'
-
-export const INDICATORS = 'INDICATOR'
-export const DATA_ELEMENTS = 'DATA_ELEMENT'
-export const DATA_SETS = 'DATA_SET'
-export const EVENT_DATA_ITEMS = 'EVENT_DATA_ITEM'
-export const PROGRAM_INDICATORS = 'PROGRAM_INDICATOR'
-export const PROGRAM_DATA_ELEMENT = 'PROGRAM_DATA_ELEMENT'
-export const PROGRAM_ATTRIBUTE = 'PROGRAM_ATTRIBUTE'
-export const DATA_ELEMENT_OPERAND = 'DATA_ELEMENT_OPERAND'
+export const DIMENSION_TYPE_ALL = 'ALL'
```

Comment:
What do you think about naming these DATA_TYPE_* ?  Just thinking out loud, not sure if it is actually better or not.